gems/aws-sdk-s3/spec/client/iad_regional_endpoint_spec.rb

```diff
@@ -63,6 +63,20 @@ module S3
             'https://s3.us-east-1.amazonaws.com/')
         end
 
+        it 'can be set through defaults mode' do
```

Comment:
Did you update all of the plugin tests?

Reply:
Yeah -  every option that has changed has a new test.